src/views/partition-table/components/Header.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <header class="section">
+    <div class="container">
+      <nav class="level is-mobile">
+        <div class="level-left">
+          <div class="level-item">
+            <h1 class="title is-size-5-mobile">
+              <strong>ESP-IDF</strong>
+              <span>Partition Table Editor</span>
+            </h1>
+          </div>
+        </div>
+        <div class="level-right">
+          <div class="level-item">
+            <p class="buttons are-small">
+              <a class="button" href="command:espIdf.flash">
+                <span class="icon is-small">
+                  <i class="fas fa-bolt"></i>
+                </span>
+                &nbsp; Flash
+              </a>
+              <button class="button" title="Retry" @click="initDataRequest">
+                <span class="icon is-small">
+                  <i class="fas fa-redo"></i>
+                </span>
+              </button>
+            </p>
+          </div>
+        </div>
+      </nav>
+      <p class="subtitle is-size-6-mobile">
+        Partition Editor will help users to easily edit partition table through
+        GUI, and quickly flash the same.
```

Comment:
This is a nice description of the "story". However, a bit weird to say "users" in third person, when the user is the one looking at the page.

I think the fact that Partition Table Editor is designed to help edit partition tables is obvious, so we can drop the line altogether?

Reply:
Updated the description, PTAL.

Since all then UIs (idf_size, heap_tracing, system_view etc) have this kind of UI, hence try to keep the subtitle to maintain a consistence look.